README.md

```diff
@@ -39,6 +39,31 @@ module "ocean-aws-k8s-vng_gpu" {
   #instance_types = ["g4dn.xlarge","g4dn.2xlarge"] # Limit VNG to specific instance types
   spot_percentage = 50 # Change the spot %
 }
+
+## Create Ocean Virtual Node Group (launchSpec) with instance_type_filters ##
+module "ocean-aws-k8s-vng" {
+  source  = "spotinst/ocean-aws-k8s-vng/spotinst"
+  ocean_id = "o-123456"
```

Comment:
use reference - module.ocean-aws-k8s.ocean_id for ocean_id

Reply:
Reference added.